src/OrchardCore.Modules/OrchardCore.Title/Views/TitlePart.Edit.cshtml

```diff
@@ -1,16 +1,21 @@
 @model TitlePartViewModel
+@using OrchardCore.ContentManagement.Metadata
+@using OrchardCore.ContentManagement.Metadata.Settings
 @using OrchardCore.ContentLocalization
 @using OrchardCore.Localization
 @using OrchardCore.Title.Models
 @using OrchardCore.Title.ViewModels;
 
+@inject IContentDefinitionManager ContentDefinitionManager
+
 @{
     var culture = await Orchard.GetContentCultureAsync(Model.TitlePart.ContentItem);
+    var displayName = ContentDefinitionManager.GetPartDefinition("TitlePart").DisplayName();
 }
 @if (Model.Settings?.Options != TitlePartOptions.GeneratedHidden)
 {
     <div class="form-group" asp-validation-class-for="Title">
-        <label asp-for="Title">@T["Title"]</label>
+        <label asp-for="Title">@displayName</label>
```

Comment:
This fix the issue, but it will break the localization in all the places where `title` is there, @sebastienros shall I fix the issue while it's make sense to react to rename title part or shall we defer this until we support data localization?

Reply:
I don't see any issue with this not being localized. I might be wrong, but isn't this one of the use-cases for dynamic localization ?

Reply:
No @jptissot I don't see issue with this either ;) but there 're many places where `title` is there